tests/pages/nodes/NodesPage-cy.js

```diff
@@ -1,97 +1,117 @@
 describe('Nodes Page', function () {
 
-  beforeEach(function () {
-    cy.configureCluster({
-      mesos: '1-for-each-health',
-      nodeHealth: true
-    });
-    cy.visitUrl({url: '/nodes'});
-  });
-
   context('Filters nodes table', function () {
 
+    beforeEach(function () {
+      cy.configureCluster({
+        mesos: '1-for-each-health',
+        nodeHealth: true
+      });
+      cy.visitUrl({url: '/nodes'});
+      cy.get('tbody tr td:first-child').as('hostnames');
+    });
+
     it('should show all nodes', function () {
-      cy.get('tbody tr').should('contain', 'dcos-01');
-      cy.get('tbody tr').should('contain', '167.114.218.155');
-      cy.get('tbody tr').should('contain', '167.114.218.156');
+      cy.get('@hostnames')
+        .should('contain', 'dcos-01')
+        .should('contain', '167.114.218.155')
+        .should('contain', '167.114.218.156');
```

Comment:
A more efficient way of running these assertions. All of the `should` calls run against the last filter call, in this case the call to `get`